### ISSUE TITLE
Accepted SPEC should have DOI

### ISSUE THREAD

**Issue by pllim** [scientific-python/specs#159]
You probably will do it but just in case...

**Comment by stefanv** [comment]
Do DOIs refer to a *specific version* of a document, or just to a URL in general? If the latter, should be feasible.

**Comment by pllim** [comment]
I think whatever works for you is fine. Though if the SPEC is going to be hard to change after being finalized and each subsequent edit is significant in itself, a separate DOI per version might be better. But otherwise, general is easier to maintain?

**Comment by stefanv** [comment]
Yes, the reason I am asking is because SPECs are defined to be *living documents*, so the idea is that they will evolve over time. The history will be in github, of course.

What is the advantage of having a DOI, other than a shorter URL?

**Comment by pllim** [comment]
I was thinking more like Zenodo for citation. Is that appropriate here? https://zenodo.org/

**Comment by stefanv** [comment]
@jarrodmillman may be better positioned to answer that question.

**Comment by MridulS** [comment]
Zenodo can indeed give out "living" DOIs, it has a base link which redirects to the latest version of a document/artifact.

**Comment by bsipocz** [comment]
This maybe interesting to be revisited. The first issue that comes in mind regarding zenodo that it track artifacts (in practice GH releases) in a repo; so I don't trivially see how to set it up for this repo where each and every SPECs should have their own separate "living" DOI as opposed to one that tracks the repo releases.

**Comment by pllim** [comment]
Yeah, over at Astropy APEs, we have to manually upload each edit via shared account, which is painful. But APE isn't supposed to change very often, so... 🤷‍♀️ 

GitHub hook via "release" would definitely be much convenient but you will need another way to track individual updates.
